src/screens/CurrencySelectModal.js

```diff
@@ -207,13 +212,13 @@ class CurrencySelectModal extends Component {
 
         filteredHigh.length &&
           filteredList.push({
-            data: filteredHigh,
-            title: filteredBest.length ? 'MORE RESULTS' : '',
+            data: filterScams(filteredBest, filteredHigh),
```

Comment:
* can we set filteredHigh to the results of filterScams before checking its length and pushing to the filtered list
* can we set the title to filteredHigh to filteredBest